department-of-veterans-affairs/va.gov-cms
# Implement DevShop YamlTasks & GitHub API CLI for Deployments

## Description

Replaces `provision-ops/yaml-tests` with `devshop/yaml-tasks`.

## Includes fixes for 
- #1229

## Testing done
- CI Tests 


## QA steps

1. Ensure tests are run and GitHub Statuses are being passed with correct links to this PR.

## Definition of Done
- [ ] Developer Release Notes
- [x] Developer Documentation has been updated.
- [ ] DevOps Repo Updates: #1303
